Makefile

```diff
@@ -55,11 +56,17 @@ PULL_BASE_REF ?= $(RELEASE_TAG) # PULL_BASE_REF will be provided by Prow
 RELEASE_ALIAS_TAG ?= $(PULL_BASE_REF)
 RELEASE_DIR := out
 
+# image name used to build the cmd/capibmadm
+TOOLCHAIN_IMAGE := toolchain
+
 TAG ?= dev
 ARCH ?= amd64
 ALL_ARCH ?= amd64 ppc64le arm64
 PULL_POLICY ?= Always
 
+# Set build time variables including version details
+LDFLAGS := $(shell ./hack/version.sh)
```

Comment:
This basically goes and sets the version in the version package but I don't see version is exist in the repository, so we might need to introduce to tag proper version with the tool we are shipping.